structuredheadings/styles/numbering.css

```diff
@@ -26,6 +26,10 @@ h6.autonumber {
 	counter-increment: h6;
 }
 
+h1.autonumber-restart {
+	counter-reset: h1;
+}
```

Comment:
Hm... I'm adding autonumber and autonumber-restart at the same time. I'm not sure how much it matters... As far as the separate element, I'm just following Spec right now :)